UserInterface.adoc

```diff
@@ -156,3 +520,6 @@ https://tex.stackexchange.com/questions/44581/creating-keyboard-layout-diagrams
 
 ISSUE:: https://github.com/zynaddsubfx/user-manual/issues/7
 ////
+
+// TODO: === Configuration window
```

Comment:
The main config is in ~/.zynaddsubfxXML.cfg if that's what you're asking about (yes, it's not in an XDG path)

Reply:
I'm talking about the window you open with the top-left Zyn button, but that's great info, thank you!

Reply:
I wouldn't mention that view too much as the view that's accessible by clicking the logo is mostly stubs instead of a fully implemented part of the UI.